### PR TITLE
Task-49720:  Add the possibility to Post or not Post on the stream

### PR DESCRIPTION
**Pull request by hassenelkamel-exo**
Prior to this change, when we post an article, a news activity is generated automatically. After this change,, we add the possibility to choose the generated news activity is posted or not posted on the stream. When we choose to not post, the generated activity is hidden.

### CODE REVIEW

File: services/src/main/java/org/exoplatform/news/NewsServiceImpl.java [hunk 1]
```diff
@@ -836,8 +838,10 @@ void postNewsActivity(News news, Session session) throws Exception {
     activity.setBody("");
     activity.setType("news");
     activity.setUserId(poster.getId());
+    activity.isHidden(news.isHiddenActivity());
     Map<String, String> templateParams = new HashMap<>();
     templateParams.put("newsId", news.getId());
+    templateParams.put("isHiddenActivity", String.valueOf(news.isHiddenActivity()));
```
Comment (azayati) [inline services/src/main/java/org/exoplatform/news/NewsServiceImpl.java]: we don't need that IMO

**Review by azayati — CHANGES_REQUESTED** [review]
See review
Keep only modifications related to post/don't post on the stream

File: services/src/main/java/org/exoplatform/news/NewsServiceImpl.java (hunk 1 above)
Comment (hassenelkamel-exo) [reply services/src/main/java/org/exoplatform/news/NewsServiceImpl.java]: yes i will remove it

File: webapp/src/main/webapp/schedule-news-drawer/components/ExoScheduleNewsDrawer.vue [hunk 2]
```diff
@@ -288,7 +289,7 @@ export default {
     disabled() {
       const postDate = new Date(this.postDate);
       const scheduleDate = new Date(this.schedulePostDate);
-      return (this.postArticleMode === 'immediate' ? false : this.postArticleMode === 'later' && postDate.getTime() === scheduleDate.getTime()) && this.selected === this.publish;
+      return (this.postArticleMode === 'immediate' ? false : this.postArticleMode === 'later' && postDate.getTime() === scheduleDate.getTime()) && this.selected === this.publish || this.stepper<3;
```
Comment (azayati) [inline webapp/src/main/webapp/schedule-news-drawer/components/ExoScheduleNewsDrawer.vue]:
```suggestion
      return (this.postArticleMode === 'immediate' ? false : this.postArticleMode === 'later' && postDate.getTime() === scheduleDate.getTime()) && this.selected === this.publish || this.stepper < 3;
```

**Review by azayati — CHANGES_REQUESTED** [review]
See review

**Review by azayati — APPROVED** [review]
LGTM after QA validation

**Pull request merged**